src/main/resources/logback.xml

```diff
@@ -0,0 +1,52 @@
+<configuration>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="AllAppender" class="ch.qos.logback.core.FileAppender">
+        <file>log/all.log</file>
+        <append>false</append>
+        <immediateFlush>true</immediateFlush>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="ModelAppender" class="ch.qos.logback.core.FileAppender">
+        <file>log/model.log</file>
+        <append>true</append>
```

Comment:
why true ?

Reply:
Honestly, I believe you don't have the right to leave such comments :""D, maybe Atef as he didn't attend the meeting at which we discussed all the details regarding the configuration file, but you? definitely not.

Anyway, I thought we don't want to get rid of the previous logs, we could use them to compare our model with the previous versions that we trained, just to pick the best parameters for our final model.

Reply:
Honestly, I believe u r right but honestly, I didn't understand anything because I never know levels in the logs or that there is a root I am tried to understand from u as much as I can. then I read the tutorial to get more context, anyway, maybe I want to write comments

Reply:
And you did and I responded :"D I am not stopping you :D write as we want but it doesn't make any sense to me to discuss things that we already agreed on you could have just stopped me and I would be more than happy to explain it to you as you always do when I am stuck on something.

Reply:
the default value is true so this is redundant also

Reply:
Done 👍